package.json

```diff
@@ -33,45 +33,46 @@
   },
   "main": "dist/ramda.js",
   "scripts": {
-    "jshint": "grunt jshint",
-    "test": "grunt jshint && grunt jscs && grunt browserify:client && grunt mochaTest:unit"
+    "browserify": "node_modules/browserify/bin/cmd.js test/*.js --outfile lib/test/bundle.js",
```

Comment:
NPM scripts automatically add `node_modules/.bin` to `$PATH` when run, so these should be able to be trimmed to:
```json
"browserify": "browserify test/*.js --outfile lib/test/bundle.js",
"clean": "rimraf dist",
"jscs": "jscs scripts/build **/*.js",
"jshint": "jshint scripts/build **/*.js",
```